src/portalbackend/PortalBackend.DBAccess/Repositories/OfferRepository.cs

```diff
@@ -350,8 +350,7 @@ sorting switch
                 o.UserRoles.Any(),
                 o.Documents.Where(doc => doc.DocumentStatusId != DocumentStatusId.LOCKED )
                     .Select(doc => new DocumentStatusData(doc.Id, doc.DocumentStatusId)),
-                o.Documents.Where(doc => doc.DocumentStatusId != DocumentStatusId.LOCKED )
-                    .Select(doc => doc.DocumentTypeId)
+                o.Documents.Select(doc => doc.DocumentTypeId)
```

Comment:
In my opinion the check needs to be changed to 
```suggestion
                o.Documents.Where(doc => doc.DocumentStatusId != DocumentStatusId.INACTIVE).Select(doc => doc.DocumentTypeId)
```

or 

doc => doc.DocumentStatusId == DocumentStatusId.ACTIVE

Reply:
fixed